crates/matrix-sdk-ui/src/sync_service.rs

```diff
@@ -79,13 +79,11 @@ struct SyncTaskSupervisor {
 }
 
 impl SyncTaskSupervisor {
-    async fn new(
-        inner: &SyncServiceInner,
-        room_list_task: JoinHandle<()>,
-        encryption_sync_task: JoinHandle<()>,
-        sender: Sender<TerminationReport>,
-        receiver: Receiver<TerminationReport>,
-    ) -> Self {
+    async fn new(inner: &SyncServiceInner) -> Self {
+        let (sender, receiver) = tokio::sync::mpsc::channel(16);
```

Comment:
Are we sure 16 is enough? Don't we want an unbounded channel, or a channel with a larger buffer?

Reply:
I think so, yeah. Basically you get only a message if one of the syncs stops or if the user stops, so it's really low volume.

In any case, this hasn't been modified from what we had previously and we didn't see any problems with this.